shared/KOSocket.cpp

```diff
@@ -139,23 +141,27 @@ bool KOSocket::DecryptPacket(uint8 *in_stream, Packet & pkt)
 
 bool KOSocket::Send(Packet * pkt) 
 {
-	if (!IsConnected() || pkt->size() + 1 > GetWriteBuffer().GetAllocatedSize())
+	if (!IsConnected()
+		|| pkt->size() + 7 > MAX_PACKET_SEND_SIZE)
 		return false;
 
-	bool r;
+	WSABUF buf;
+	DWORD w_length = 0;
+	DWORD flags = 0;
+	BYTE out_stream[MAX_PACKET_SEND_SIZE];
+	BYTE Final_Packet[MAX_PACKET_SEND_SIZE];
+	uint32 Final_Size = 0;
+	memset(out_stream, 0x00, MAX_PACKET_SEND_SIZE);
+	memset(Final_Packet, 0x00, MAX_PACKET_SEND_SIZE);
```

Comment:
```suggestion
	memset(out_stream, 0, MAX_PACKET_SEND_SIZE);
	memset(Final_Packet, 0, MAX_PACKET_SEND_SIZE);
```

Also a rename from:
* `out_stream` -> `byOutStream`
* `Final_Packet` -> `byFinalPacket`